File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        tracker = {}
        for i in range(len(nums)):
            if nums[i] not in tracker:
                tracker[nums[i]] = 0
            tracker[nums[i]] += 1
        sort_frequency = sorted(tracker.items(), key = lambda x: x[1], reverse = True)
        sol = []
        for j in range(k):
            sol.append(sort_frequency[j][0])
        return sol

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array-ii/submission-1.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int l = 0;
        int r = 0;
        int index = 0;
        int count = 0;
        while(r < nums.length){
            if(nums[r] == nums[l]){
                if(count < 2){
                    count++;
                    nums[index] = nums[r];
                    index++;
                }
            }
            else{
                count = 1;
                nums[index] = nums[r];
                l = r;
                index++;
            }
            r++;

        }
        return index;
    }
}